lib/authy/onetouch.rb

```diff
@@ -23,7 +23,7 @@ def self.send_approval_request(params)
       begin
         self.clean_hash!(details)
         self.clean_hash!(hidden_details)
-        self.clean_logos!(logos)
+        logos = self.clean_logos!(logos)
```

Comment:
Yeah, this wasn't working before because `self.clean_logos!` wasn't actually affecting the overall state. It turned out that even if you set logos in your arguments to the method, nothing was ever sent to the API. This fixes that.